scripts/core/editor3/html/to-html/HTMLGenerator.js

```diff
@@ -29,11 +31,12 @@ const BlockStyleTags = {
  * tables.
  */
 export class HTMLGenerator {
-    constructor(contentState, disabled = []) {
+    constructor(contentState, editorState, disabled = []) {
```

Comment:
I think `contentState` you already had is enough for generating HTML. `editorState` is the same as `contentState` except it has a `selectionState` which you will no be using for HTML generation anyway.